### PR TITLE
Mobile  menu

### PR DESCRIPTION
**Pull request by osamaashraf6**
## What does this PR do?

This PR is raised to implement the following features: 👍 :

- [x] When the user clicks (or taps) the hamburger button, the mobile menu appears.
- [x] When the user clicks (or taps) the close (X) button, the mobile menu disappears.
- [x] When the user clicks (or taps) any of the mobile menu options, the mobile menu disappears.
- [x] When the user clicks (or taps) any of the mobile menu options, the correct part of the page is displayed.

## Description of the task to be completed 👎 

- [x] Add the HTML for the structure. 
- [x] Add the CSS file for the styling.
- [x] Use the linter tool that analyzes my source code to flag programming errors, bugs, stylistic errors, and suspicious constructs. 💯 
- [x] Use Figma a collaborative UI design tool built in the browser. 
- [x] Use Flexbox a layout tool in CSS that helps you easily position HTML elements on the page.
- [x] Check the accessibility of this portfolio which means that websites, tools, and technologies are designed and developed so that people with disabilities ♿ can use them. For more information about Web accessibility [Click here](https://github.com/microverseinc/curriculum-html-css/blob/main/accessibility.md)☺️
     

  
## How should it be tested

Clone the repository
```
git clone https://github.com/osamaashraf6/portfolio.git
```
Navigate to the folder
```
cd portfolio
```
Checkout the branch
```
git checkout mobile__menu
```

#### Thanks for reviewing my PR 😊

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -32,18 +32,16 @@
...
               </li>
               <li>
-                <ul class="iwq">
-                  <li><a href="#">Portoflio</a></li>
-                  <li><a href="#">About</a></li>
-                  <li><a href="#">Contact</a></li>
+                <ul class="iwq ">
+                  <li><a class="mobile-nav-links" herf="#">Portoflio</a></li>
+                  <li><a class="mobile-nav-links" href="#">About</a></li>
+                  <li><a class="mobile-nav-links" href="#">Contact</a></li>
```
Comment (Abenezer-Tilahun) [inline index.html]: - [ ] As stated in the project requirement please make sure that When the user clicks (or taps) any of the mobile menu options, the correct part of the page is displayed and also correct the type spelling for "href" . Kindly work on it.

**Review by Abenezer-Tilahun — CHANGES_REQUESTED** [review]
Hi @osamaashraf6  , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights 👏🏻

✔️ No linter errors.
✔️ You used correct GitHub Flow.
✔️ Professional   README
✔️ Good commit messages

almost

## Required Changes ♻️

_Check the comments below the review._

## Optional suggestions


_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (osamaashraf6) [reply index.html]: -  Thanks 🙏for your review I have noticed what you told me to modify and I will work on it.😊

File: index.html [hunk 2]
```diff
@@ -39,9 +39,9 @@
               </li>
               <li>
                 <ul class="iwq ">
-                  <li><a class="mobile-nav-links" herf="#">Portoflio</a></li>
-                  <li><a class="mobile-nav-links" href="#">About</a></li>
-                  <li><a class="mobile-nav-links" href="#">Contact</a></li>
+                  <li><a class="mobile-nav-links" href="#portfolio">Portoflio</a></li>
```
Comment (ichala) [inline index.html:42]: - [ ] your website looks fantastic I would love to suggest adding a smoother animation to your website's scroll behavior, try to add `scroll-behavior: smooth;` to your HTML tag in CSS it works like magic! 😮 

``` css
html {
scroll-behavior: smooth;
}
```

Before           | After
:-------------------------:|:-------------------------:
before | mAFTER

**Review by ichala — APPROVED** [review]
Hi @osamaashraf6 ,
# Status: Approved :heavy_check_mark: :heavy_check_mark: :heavy_check_mark:  

 
68747470733a2f2f6d65646961322e67697068792e636f6d2f6d656469612f6c3366516333614356304871707a5763552f67697068792e676966

# To Highlight 💪🏻

- Changes Implemented✔️  

 Good job implementing the required changes :+1: Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

# Optional Suggestions  :

- [ ] **_Check the comment Below_**


**********
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag  @ichala in your question so I can receive the notification.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 2 above)
Comment (osamaashraf6) [reply index.html:42]: - Thanks for your review 😊I already made the page moves by scroll behavior,  maybe you didn't notice that you can check this image to make sure from what I am saying. 👎 


Screenshot 2022-10-12 180123

**Pull request merged**